seaclouds-catalog.bom

```diff
@@ -13,10 +13,11 @@ brooklyn.catalog:
         planner.url: https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=planner-service&v=LATEST&e=jar
         planner.config.port: 1234
         planner.config.discoverer.url: undefined
-        planner.config.optimizer.url: undefined
+        planner.config.optimizer.url: $brooklyn:formatString("http://%s:%s", component("optimizer-service").attributeWhenReady("host.address"), component("optimizer-service").attributeWhenReady("http.port"))
```

Comment:
This is the default value of the entity, I would rather set a default value like localhost:1234 and then, pass the brooklyn.format.... when you add the entity to the byon. If you want to see an example, you could take a look to the dashboard section in the bom.